src/caches/sdirk_caches.jl

```diff
@@ -549,3 +549,50 @@ function alg_cache(alg::Union{Hairer4,Hairer42},u,rate_prototype,uEltypeNoUnits,
   Hairer4Cache(u,uprev,du1,fsalfirst,k,z₁,z₂,z₃,z₄,z₅,dz,b,tmp,atmp,J,
                W,uf,jac_config,linsolve,nlsolve,tab)
 end
+
+@cache mutable struct MEBDFCache{uType,rateType,uNoUnitsType,JType,WType,UF,JC,F,N} <: SDIRKMutableCache
```

Comment:
I wouldn't classify it as SDIRK. It should go in the bdf files.

Reply:
true, it fits better into this category. I have moved it over and did the other updates aswell.